boilerup-wp.php

```diff
@@ -110,7 +111,136 @@ public static function add_segment_form_identify() {
 
             <?php
         }
+        public static function add_segment_body_code() {
+            ?>
+            
+            <script>
+                var segment_purdue = {
+                    formSubmitted: function(event){
+                            
+                        event.preventDefault();
+                        // First, call the identify event and include any important user profile trait values
+                        let traits = {
+                            first_name : document.getElementById('form-first_name').value,
+                            last_name : document.getElementById('form-last_name').value,
+                            email : document.getElementById('form-email').value,
+                            address : {
+                                street : document.getElementById('form-street').value,
+                                city : document.getElementById('form-city').value,
+                                phone : document.getElementById('form-phone').value,
+                                postalCode : document.getElementById('form-zip_code').value,
+                                country : document.getElementById('form-form-country').value
+                            }
```

Comment:
This is all example code that won't actually collect anything from Gravity forms, can you either comment this out or add this as a feature to this pr. Some of the things were already defined by me in my test code before hand you can use that as an example or you can just comment this function out for now and we can work on it in a separate PR. Thanks!

Reply:
Right. I deleted the invocation of the formSubmitted function but left it there. I was thinking maybe we could just modify this function later for the tracking of the form and the identification. I'll delete it on this branch and work on the form on a different branch.